ush/config_defaults.yaml

```diff
@@ -2427,12 +2427,40 @@ verification:
   OBS_NDAS_SFCorUPA_FN_TEMPLATE: 'prepbufr.ndas.{valid?fmt=%Y%m%d%H}'
   OBS_NDAS_SFCorUPA_FN_METPROC_TEMPLATE: '${OBS_NDAS_SFCorUPA_FN_TEMPLATE}.nc'
   #
+  # RUN_VX_DET:
+  # Flag that specifies whether to run deterministic verification.  If set
+  # to True, this will run deterministic vx on the post-processed forecast
+  # output.  This post-processed output may consist of a single forecast
+  # or an ensemble of foreasts, and it may be staged from previous runs of
+  # the SRW App or may be generated by running the TN_RUN_FCST task as part
+  # of the current SRW-App-generated experiment.
+  #
+  # RUN_VX_ENS:
+  # Flag that specifies whether to run ensemble verification.  The ensemble
+  # forecast output on which vx will be run may be staged or generated by
+  # running an ensemble of forecasts with the weather model as part of the
+  # current SRW-App-generated experiment.
+  #
+  RUN_VX_DET: False
+  RUN_VX_ENS: False
+  #
```

Comment:
In general, I think we should avoid including these types of workflow switches. Perhaps it makes sense to break out the PcpCombine* and Pb2nc* task so that it's easier to specify running those. Then all the user has to do is specify which groups they'd like to run instead of using these flags to turn off a group of items.

For example, since we have these flags, the user needs to set two different things in the config file to even turn on ensemble verification...the task group must be added AND the RUN_VX_ENS flag must be set to True. If these flags were removed, the only thing they'd need to do is include the verify_ens.yaml file in task groups.

Since I likely misunderstood the assignment when grouping the PcpCombine* and Pb2nc* into the verify_det.yaml group, and those may also be needed when running ensemble vx, maybe they belong in their own taskgroup like `prepare_vx_obs` or something similar. Then the user chooses to turn these tasks on/off by including/not including them in the task group instead of having this second switch that will give un-intended results. 

Mainly what I'm concerned with is a user saying "I turned on this switch, why is it not working?" and them also needing to know that they must flip another switch.

Reply:
@christinaholtNOAA Yes, somehow I got it stuck in my head that I shouldn't further split the `verify_[det|ens].yaml` files, but that's the best solution to avoiding these flags.  It was also bothering me that the user has to change two things in the config file to get vx to work right.  Thanks for the suggestion, I'll change it.

Reply:
@christinaholtNOAA Now the tasks that are prerequisites for both deterministic and ensemble vx are in a yaml file of their own (`verify_pre.yaml`), and the `RUN_VX_[DET|ENS]` switches are removed from `config_defaults.yaml`.